packages/flutter/lib/src/cupertino/desktop_text_selection_toolbar.dart

```diff
@@ -74,6 +74,14 @@ class CupertinoDesktopTextSelectionToolbar extends StatelessWidget {
           color: _kToolbarBorderColor.resolveFrom(context),
         ),
         borderRadius: const BorderRadius.all(_kToolbarBorderRadius),
+        boxShadow: const <BoxShadow>[
```

Comment:
Nit: Can you abstract this shadow to a const variable and put it at the top with the others?  Mainly I say that just so that it's clear that these values came from the same MacOS 13.2 screenshot as the other values.

Reply:
Don't know how I didn't think of this, I will place it with the other constants :smile: